doc/classes/ProjectSettings.xml

```diff
@@ -317,6 +317,9 @@
 			If [code]true[/code], disables printing to standard output. This is equivalent to starting the editor or project with the [code]--quiet[/code] [url=$DOCS_URL/tutorials/editor/command_line_tutorial.html]command line argument[/url]. See also [member application/run/disable_stderr].
 			Changes to this setting will only be applied upon restarting the application.
 		</member>
+		<member name="application/run/print_header" type="bool" setter="" getter="" default="true">
+			If set to [code]false[/code], disables printing engine header on startup.
```

Comment:
It's more common in the class reference to talk about "What happens if **enabled**", regardless of the default value. It's also not a bad idea to be more explicit here.
```suggestion
			If set to [code]true[/code], the engine header is printed in the console on startup. This header describes the current version of the engine, as well as the renderer being used.
```